murdock.py

```diff
@@ -145,6 +145,14 @@ def __init__(s, data):
         s.labels = set()
         s.old_head = None
 
+        status = {
+                "state": "pending",
+                "description": "Waiting for review",
+                "context": config.context
+                }
+        s.set_status(s.head, "pending", status)
```

Comment:
This (a little different) should be done in the handler when the "opened" event triggers (line 4xx). But it doesn't. I'll look into it.

Doing it here is a bad idea, as doing things in a constructor which might throw exceptions must be handled, which I don't usually.